.github/workflows/deploy-frontends-to-production.yml

```
@@ -222,6 +226,92 @@ jobs:
               gcloud app versions delete "$version" --service=$service --quiet
             done
 
+  ### deploy website2 ###
+  website2:
+    name: build-push-deploy-website2
+    needs: [image-tag]
+    if: inputs.website2 == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)